packages/vm/src/evm/opcodes/functions.ts

```diff
@@ -888,6 +888,9 @@ export const handlers: Map<number, OpHandler> = new Map([
   [
     0xb4,
     function (runState) {
+      if (runState.eei.isStatic()) {
```

Comment:
realizing the implementation should also be updated to use the warm SSTORE of a dirty slot gas schedule instead of a fixed 100 gas (and same with SLOAD)